### PR TITLE
Create configMap to nginx deploy, and only deploy nginx if nginx.conf

### PR DESCRIPTION
**Pull request by yagonobre**
👶

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [NEXT_RELEASE]
 ### Changed
 - Change Method `CreateSecret` to `CreateOrUpdateSecret` on k8s interfaces
+- Only deploy nginx if nginx.conf exist
+- Create configmap on deploy with nginx
```
Comment (aguerra) [inline CHANGELOG.md]: The feature hasn't been released yet so I think you shouldn't add these lines. What do you guys think?

Comment (yagonobre) [reply CHANGELOG.md]: LGTM

Comment (drgarcia1986) [reply CHANGELOG.md]: Agree

File: pkg/server/k8s/client.go [hunk 2]
```diff
@@ -440,7 +440,7 @@ func (k *Client) Limits(namespace, name string) (*app.Limits, error) {
 	return lim, nil
 }
 
-func (k *Client) createOrUpdateConfigMap(namespace, name string, data map[string]string) error {
+func (k *Client) CreateOrUpdateConfigMap(namespace, name string, data map[string]string) error {
```
Comment (drgarcia1986) [inline pkg/server/k8s/client.go]: 💅  could you change receiver name to `c`?

**Pull request merged**